samcli/lib/cli_validation/image_repository_validation.py

```diff
@@ -103,6 +82,17 @@ def wrapped(*args, **kwargs):
                         option_name="--image-repositories", ctx=ctx, message=image_repos_error_msg
                     ),
                 ),
+                Validator(
+                    validation_function=lambda: not guided
+                    and not image_repositories
+                    and not _is_all_image_funcs_provided(template_file, {}, parameters_overrides)
+                    and not (image_repository or resolve_image_repos),
```

Comment:
why not to check all parameters in one condition like and `not (image_repositories or image_repository or resolve_image_repos)` instead of having them in 2 conditions?